addons/mail/static/src/components/composer/composer.scss

```diff
@@ -161,6 +158,14 @@
 // Style
 // ------------------------------------------------------------------
 
+// TODO FIXME o-open on the button should be enough
```

Comment:
add dot at the end, because it looks like the rest is the same phrase otherwise and it doesn't read well